pages/play.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div class="flex flex-column justify-between items-center tc vh-100">
+    <SpectrumGame :turns="playTurns" :next-path="nextPath" />
+  </div>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      title: 'Play',
+      playTurns: [
+        {
+          object: 'Boudica',
+          spectrum: ['Submissive', 'Dominant'],
+        },
+        {
+          object: 'Roman Empire',
+          spectrum: ['Proud', 'Modest'],
+        },
+        {
+          object: "Hadrian's Wall",
+          spectrum: ['Generous', 'Greedy'],
+        },
+      ],
+      nextPath: '/future',
+    }
+  },
+}
+</script>
+
+<style lang="scss">
+    @import '~assets/scss/font-families.scss';
+    @import '~assets/scss/global.scss';
```

Comment:
I thought these global styles were dealt with in nuxt.config.js ?
please remove here and in other files :)

Reply:
I don't know how this came back, I pulled from from main and merged into this branch before this PR. I'm fixing it